flycheck.el

```diff
@@ -6777,18 +6777,14 @@ Each item is a string with a tag to be given to `go build'."
 (flycheck-define-checker go-build
   "A Go syntax and type checker using the `go build' command.
 
+Requires Go 1.6 or newer.
+
 See URL `https://golang.org/cmd/go'."
```

Comment:
Reminds me of a recent-ish thread on Emacs-devel that led to https://github.com/cpitclaudel/compact-docstrings . I had a similar patch for doc buffers there, but nobody seemed to care too much.